From 5b87ac2b86c3fbdddbcfb070f2c6f15580d6bfe2 Mon Sep 17 00:00:00 2001
From: Vanessa Maegima <vanessa.maegima@foundries.io>
Date: Tue, 6 Jul 2021 09:41:39 -0300
Subject: [PATCH] configs: imx8mm_var_dart: avoid config fallback error

CONFIG_SYS_I2C cannot be defined when CONFIG_DM_I2C is used, so handle
these configs in the same way as the other imx8mm boards.

Signed-off-by: Vanessa Maegima <vanessa.maegima@foundries.io>

---
 include/configs/imx8mm_var_dart.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/imx8mm_var_dart.h b/include/configs/imx8mm_var_dart.h
index 0045663f33..ea6eebc33e 100644
--- a/include/configs/imx8mm_var_dart.h
+++ b/include/configs/imx8mm_var_dart.h
@@ -34,6 +34,7 @@
 #define CONFIG_POWER
 #define CONFIG_POWER_I2C
 #define CONFIG_POWER_BD71837
+#undef CONFIG_DM_I2C
 
 #define CONFIG_SYS_I2C
 
